boards/arm64/xenvm/board.cmake

```diff
@@ -3,13 +3,14 @@ set(QEMU_ARCH aarch64)
 
 set(QEMU_CPU_TYPE_${ARCH} cortex-a57)
 
+set(QEMU_GIC_VERSION 3)
 set(QEMU_KERNEL_OPTION -kernel ${ZEPHYR_BINARY_DIR}/xen)
 set(QEMU_FLAGS_${ARCH}
    -gdb tcp::4224
    -cpu ${QEMU_CPU_TYPE_${ARCH}} -m 6G
    -nographic
```

Comment:
I think we still need to make GIC verision optional. I see this implementation as to create separate configs to handle both gicv2 and gicv3